_data/doc_config.json

```diff
@@ -1,6 +1,9 @@
 {
-  "name": "SigV4-for-AWS-IoT-embedded-sdk",
-  "releases": [],
+  "name": "AWS SigV4 Library",
+  "releases": [
+    "v1.1.0",
```

Comment:
Could this be auto-generated based on tags?

Reply:
This is automatically updated in the future though GH Actions. This is a one time thing to set initial state.